mappings/net/minecraft/recipe/Ingredient.mapping

```diff
@@ -18,6 +18,8 @@ CLASS net/minecraft/class_1856 net/minecraft/recipe/Ingredient
 		ARG 0 items
 	METHOD method_8092 ofEntries (Ljava/util/stream/Stream;)Lnet/minecraft/class_1856;
 		ARG 0 entries
+	METHOD method_8093 test (Ljava/lang/Object;)Z
```

Comment:
This descriptor seems wrong. From my extracted class_1856 the descriptor of method_8093 according to javap is `public boolean method_8093(net.minecraft.class_1799);`, so this should be
```suggestion
	METHOD method_8093 test (Lnet/minecraft/class_1799;)Z
```
Dunno how your enigma get this wrong.